testdata/column.input.txt

```diff
@@ -0,0 +1,9 @@
+60916 s003  Ss+    0:00.51 /bin/bash -l
```

Comment:
From a testing perspective this file should also contain some other white space characters like `\t` and maybe even a mix if spaces and `\t`/

Reply:
Agreed. On the other hand, since we're using `strings.Fields`, we don't want to fall into the trap of testing `strings.Fields`. I'll see what I can do to make this test a little more fiendish.

Reply:
I understand your point, but in my opinion the usage of `strings.Fields` is an implementation detail. If you decide to adjust the documentation comment of the function `Column` about what is considered white space, then this becomes the API surface of this function, which in my opinion should be tested regardless of the specific implementation.

Reply:
Agreed and done.